ros/src/sensing/filters/packages/image_processor/CHANGELOG.rst

```diff
@@ -0,0 +1,7 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package image_preprocessor
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1.6 (2017-11-15)
+------------------
+Initial release
```

Comment:
@amc-nu  you do not need to do this, this will be done by a release script: https://github.com/CPFL/Autoware/issues/766#issuecomment-325096495 (step 3., `catkin_prepare_release`).

Reply:
Good to know! Thanks!